addons/pulsar/scripts/init-broker.sh

```diff
@@ -85,11 +85,6 @@ init_broker() {
   fi
 
   initialize_cluster_metadata "$clusterName" "$zookeeperServers" "$webServiceUrl" "$brokerServiceUrl"
-  quit_script
-}
-
-quit_script() {
```

Comment:
This function still appears in the file (a few lines above).

Reply:
> This function still appears in the file (a few lines above).

fixed done